app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -542,41 +548,45 @@ def process_enrollment_response(enrollment, response)
     end
   end
 
-  def send_verified_email(user, enrollment)
+  def send_verified_email(user, enrollment, visited_location_name)
```

Comment:
- I think positional arguments only make sense when it's clear from the callsite what the argument's purpose is. I don't think it's particularly clear what a `'none'` third argument is meant to do from looking at it in isolation, for example. Refactoring to use keyword arguments would be my preference.
- I see that we pass `user` as `enrollment.user` in all cases, but since we're already passing `enrollment`, it seems unnecessary to have it as an argument if the method could reference the user directly from `enrollment.user`.
- On a related note about collapsing arguments to use `enrollment`, I'd wonder if `visited_location_name` is something we'd _want_ to store on the enrollment, if we're not already?

Reply:
We save the selected location but not the visited location.  I had this same thought but wondered outside of these emails is it needed. Maybe nice in analytics so would make that easier.

Reply:
- I agree about removing `user` as an argument method.  I will make that change. 
- Storing the `visited_location_name` on the `in_person_enrollment` in the database is a bigger change.  I'd like to hear what the team thinks and will follow up in Joy eng huddle.